stable/kuberhealthy/templates/deployment.yaml

```diff
@@ -28,6 +28,9 @@ spec:
       {{- end }}
       {{- if .Values.prometheus.enabled -}}
       {{- if .Values.prometheus.enableScraping -}}
+      {{- if not .Values.deployment.podAnnotations }}
```

Comment:
if `.Values.deployment.podAnnotations` is empty, then put in `annotations:`?

Maybe what you really want here is a static annotation that always exists (to make your life easier), then some `if` statements around the specific `prometheus.io/scrape` annotations?

Reply:
Because if there are no specified annotations, but scraping is enabled, we need the annotations field for that. I don't want the static annotation field in the case where no annotation sare provided, and the prometheus.enabled is set to false

Reply:
k